<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="outputlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>4</number>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="margin">
     <number>20</number>
    </property>
   </widget>
   <widget class="QPushButton" name="zeroButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ceButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="divButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="threeButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="oneButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="twoButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sixButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>410</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fourButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fiveButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>410</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minusButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>410</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sevenButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plusButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>560</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eigthButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>560</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nineButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>560</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="equalButton">
    <property name="geometry">
     <rect>
      <x>309</x>
      <y>10</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pointButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>110</y>
      <width>80</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans Narrow</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>486</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
